packages/nimble-components/src/theme-provider/design-tokens.ts

```diff
@@ -664,6 +666,90 @@ export const largeDelay = DesignToken.create<string>(
+).withDefault('1px');
+
+export const bannerButtonHeight = DesignToken.create<string>(
+    styleNameFromTokenName(tokenNames.bannerButtonHeight)
+).withDefault('24px');
+
+export const bannerFontSize = DesignToken.create<string>(
+    styleNameFromTokenName(tokenNames.bannerFontSize)
+).withDefault('12.8px');
+
+export const bannerColor = DesignToken.create<string>(
+    styleNameFromTokenName(tokenNames.bannerColor)
+).withDefault(White);
+
+export const bannerNeutralBackgroundColor = DesignToken.create<string>(
+    styleNameFromTokenName(tokenNames.bannerNeutralBackgroundColor)
+).withDefault((element: HTMLElement) => getColorForTheme(element, Black75, Black75, hexToRgbaCssColor(White, 0.3)));
+
+export const bannerErrorBackgroundColor = DesignToken.create<string>(
+    styleNameFromTokenName(tokenNames.bannerErrorBackgroundColor)
+).withDefault((element: HTMLElement) => getColorForTheme(
+    element,
+    Fail100LightUi,
+    BannerFail100DarkUi,
+    hexToRgbaCssColor(White, 0.3)
+));
+
+export const bannerWarningBackgroundColor = DesignToken.create<string>(
+    styleNameFromTokenName(tokenNames.bannerWarningBackgroundColor)
+).withDefault((element: HTMLElement) => getColorForTheme(
+    element,
+    Warning100LightUi,
+    Warning100DarkUi,
+    hexToRgbaCssColor(White, 0.3)
+));
+
+export const bannerInfoBackgroundColor = DesignToken.create<string>(
+    styleNameFromTokenName(tokenNames.bannerInfoBackgroundColor)
+).withDefault((element: HTMLElement) => getColorForTheme(
+    element,
+    Information100LightUi,
+    Information100DarkUi,
+    hexToRgbaCssColor(White, 0.3)
+));
+
+export const bannerIconColor = DesignToken.create<string>(
+    styleNameFromTokenName(tokenNames.bannerIconColor)
+).withDefault(hexToRgbaCssColor(White, 0.6));
+
+export const bannerIconMarginTop = DesignToken.create<string>(
+    styleNameFromTokenName(tokenNames.bannerIconMarginTop)
+).withDefault('8px');
+
+export const bannerStartEndWidth = DesignToken.create<string>(
+    styleNameFromTokenName(tokenNames.bannerStartEndWidth)
+).withDefault('48px');
+
+export const bannerTextMarginVertical = DesignToken.create<string>(
+    styleNameFromTokenName(tokenNames.bannerTextMarginVertical)
+).withDefault('7px');
+
+export const bannerTextGap = DesignToken.create<string>(
+    styleNameFromTokenName(tokenNames.bannerTextGap)
+).withDefault('8px');
+
+export const bannerButtonFillSelectedColor = DesignToken.create<string>(
+    styleNameFromTokenName(tokenNames.bannerButtonFillSelectedColor)
+).withDefault(hexToRgbaCssColor(White, 0.2));
+
+export const bannerLinkActiveFontColor = DesignToken.create<string>(
+    styleNameFromTokenName(tokenNames.bannerLinkActiveFontColor)
+).withDefault(hexToRgbaCssColor(White, 0.6));
+
+export const bannerCloseButtonSize = DesignToken.create<string>(
+    styleNameFromTokenName(tokenNames.bannerCloseButtonSize)
+).withDefault('16px');
+
+export const bannerCloseButtonIconSize = DesignToken.create<string>(
```

Comment:
> These all seem extremely specific to banners,

I agree. I wasn't thinking about these tokens in terms of being part of our public API. I was just trying to avoid having a bunch of constants in the style sheet. Maybe that's actually the better option, though. I could use some input/guidance on how we want to handle these banner-specific values.